docs/user/dashboard/make-dashboards-interactive.asciidoc

```diff
@@ -23,78 +23,125 @@ data-type="inline"
 </br>
 ++++
 
-Third-party developers can create drilldowns. To learn how to code drilldowns, refer to {kib-repo}blob/{branch}/x-pack/examples/ui_actions_enhanced_examples[this example plugin].
-
-[float]
 [[add-controls]]
-=== experimental[] Add Controls panels
+=== Filter dashboard data with controls
+
+experimental[] *Controls* are interactive panels you add to your dashboards to filter and display only the data you want to explore.
 
-*Controls* panels allow you to apply dashboard-level filters based on values from a list, or a range of values. 
+deprecated::[8.3.0, "link:https://www.elastic.co/guide/en/kibana/8.1/drilldowns.html#add-controls[*Input controls*] are deprecated in 8.3.0, and will be removed in a future release. To prepare for the removal of input controls, add the new controls to your dashboards."]
 
-There are two types of *Controls* you can add to dashboards:
+There are two types of controls:
 
-* *Options list* &mdash; Filters content based on one or more specified options. The dropdown menu is dynamically populated with the results of a terms aggregation. 
-For example, use the options list on the sample flight dashboard when you want to filter the data by origin city and destination city.
+* *Options list* &mdash; Adds a dropdown that allows you to filter the data with one or more options that you select.
++
+For example, if you are using the *[Logs] Web Traffic* dashboard from the sample web logs data, you can add an options list for the `machine.os.keyword` field that allows you to display only the logs that generated from `osx` and `ios` operating systems.
++
+[role="screenshot"]
+image::images/dashboard_controlsOptionsList_8.3.0.png[Options list control for the `machine.os.keyword` field with the `osx` and `ios` options selected]
 
-* *Range slider* &mdash; Filters data within a specified range of numbers. The minimum and maximum values are dynamically populated with the results of a 
-min and max aggregation. For example, use the range slider when you want to filter the sample flight dashboard by a specific average ticket price.
+* *Range slider* &mdash; Adds a slider that allows you to filter the data within a specified range of values. 
++
+For example, if you are using the *[Logs] Web Traffic* dashboard from the sample web logs data, you can add a range slider for the `hour_of_day` field that allows you to display only the log data from 9:00AM to 5:00PM.
 +
 [role="screenshot"]
-image::images/dashboard-controls.png[]
+image::images/dashboard_controlsRangeSlider_8.3.0.png[Range slider control for the `hour_of_day` field with a range of `9` to `17` selected]
 
-To create *Controls* panels:
+[float]
+[[create-and-add-controls]]
+==== Create and add controls
+
+To add interactive filters, create controls, then add them to your dashboard.
+
+. Open or create a new dashboard. 
+
+.. If you are adding a *Control* to an existing dashboard, click *Edit*, then click *Add control* on the dashboard.
```

Comment:
I'm not sure if we should direct them to create controls through the callout necessarily, since this is primarily meant to draw attention to the new controls and it will eventually be removed and/or dismissed by the user. 

Maybe we could just always tell them to use the toolbar `Controls` button, even though they could technically use the callout too? Open to thoughts though :)